sound/soc/intel/boards/sof_da7219_max98373.c

```diff
@@ -331,6 +375,16 @@ static int audio_probe(struct platform_device *pdev)
 	if (!ctx)
 		return -ENOMEM;
 
+	/* By default dais[0] is configured for max98373 */
+	if (!strcmp(pdev->name, "sof_da7219_max98360a")) {
+		dais[0] = (struct snd_soc_dai_link) {
+			.name = "SSP1-Codec",
+			.no_pcm = 1,
+			.dpcm_playback = 1,
+			.ignore_pmdown_time = 1,
+			SND_SOC_DAILINK_REG(ssp1_pin, dummy) };
```

Comment:
should it be SND_SOC_DAILINK_REG(ssp1_pin, dummy, platform) ?

what you want is a dummy codec, yes?

Reply:
Yes, that's correct, I dropped the "platform" for aesthetic as the platform_name will be fixed up.

Reply:
but not having a platform here results in no allocation of the memory, we've seen this before. the fixup will change the name, but it requires a platform to be defined initially.

Reply:
In this driver, the dai[] is statically allocated already, that said, let me add the platform parameter and .id to make it explicit and consistent with other DAI links.

Reply:
Updated https://github.com/thesofproject/linux/commit/0bccc6157ecf26648a5d639c83b1a3575ef0777d

ver 5: add platform to SND_SOC_DAILINK_REG(), also add .id=0 to make it explicit set.